helpers/makePriceRequestsByKey.ts

```diff
@@ -29,7 +29,7 @@ function formatPriceRequest(priceRequest: RawPriceRequestDataT) {
   const decodedIdentifier = decodeHexString(identifier);
   const decodedAncillaryData = decodeHexString(ancillaryData);
   const correctVote = priceRequest.correctVote;
-  const uniqueKey = makeUniqueKeyForVote(identifier, time, ancillaryData);
+  const uniqueKey = makeUniqueKeyForVote(decodedIdentifier, time, ancillaryData);
```

Comment:
the subgraph makes unique keys for votes with the decoded identifier, so I changed the implementation on the frontend to do the same. This also makes finding values in the devtools much easier.